### PR TITLE
Create an event making long press on any place of the map

### PR DESCRIPTION
**Pull request by frabago**
- Get coordinates where the user make a longPress
- Display a blue container where the user generate an event
- Display the new event modal
- Associate the modal with the createEvent backend service

There is an issue when the user make a longPress and doesn't confirm the event, in this case the container icon is generated as well. We should make a fix for this issue in other PR

### CODE REVIEW

File: src/actions/GatherActions.js [hunk 1]
```diff
@@ -16,6 +16,11 @@ export const actionTypes = {
   GET_CONTAINERS_SUCCESS: 'GET_CONTAINERS_SUCCESS',
   GET_CONTAINERS_ERROR: 'GET_CONTAINERS_ERROR',
   SET_CONTAINER_ID: 'SET_CONTAINER_ID',
+  CREATE_EVENT_REQUEST: 'CREATE_EVENT_REQUEST',
+  CREATE_EVENT_SUCCES: 'CREATE_EVENT_SUCCES',
```
Comment (anitanigma) [inline src/actions/GatherActions.js]: Success *

File: src/components/Gather/Gather.js [hunk 2]
```diff
@@ -147,6 +155,13 @@ class Gather extends Component {
     });
   };
 
+  createExtraEvent = (e) => {
+    this.toggleAddEventModal();
+    this.props.setEventCoordinates(e.geometry.coordinates);
+    this.setState({ eventCoordinates: e.geometry.coordinates });
+    this.setState({ showEvents: true });
```
Comment (anitanigma) [inline src/components/Gather/Gather.js]: Any particular reason why these two are in different lines?

**Comment by anitanigma** [comment]
List of a few fixes/improvements:

1. If you click outside the modal without adding an event the icon is still shown in the map.
2. Change 'Description of the event' to 'Event description'.

**Comment by dsusviela** [comment]
String has already been change @anitanigma

**Pull request merged**